front/src/common/redux/services/sudentRegistrationService.js

```diff
@@ -0,0 +1,18 @@
+async function studentRegistrationService(student) {
```

Comment:
Call it something like `registerStudentApi`. Service is not an appropriate name for function because usually service is an actual object

Reply:
Renamed to registerStudentApi